apps/system/js/audio_channel_service.js

```diff
@@ -122,10 +122,11 @@
         if (!isBackground) {
           this.publish('visibleaudiochannelchanged', channel);
         }
-        this.publish('audiochannelchangedasactive', channel);
+        this.publish('audiochannelchanged', channel);
       } else {
         this._resetAudioChannel(audioChannel);
         this._resumeAudioChannels();
+        this.publish('audiochannelchanged', { channel: 'none' });
```

Comment:
@evanxd This line should be the fix of the root cause, right? for the others I think we just change the event names from `audiochannelchangedasactive` to  `audiochannelchanged`.